setup.py

```diff
@@ -30,7 +30,7 @@ def read(fname):
         'six',
         'pytz',
         'unidecode>=0.04.13',
-        'django_autoslug',
+        'django-autoslug-iplweb==1.9.4.dev0',
```

Comment:
cant we have >=1.9,<1.10 ?

Reply:
This is a fork by @mpasternak that has only one version for now. It's a nice alternative until the autoslug package is fixed.

Reply:
Sounds good, @mpasternak feel free to move the repo to the org, create a team and add @wswld if you feel like it